randovania/games/am2r/json_data/The Tower.txt

```diff
@@ -323,7 +335,7 @@ Extra - map_name: rm_a4h05
   > Dock to Tower Exterior South East
       Any of the following:
           Space Jump
-          Hi-Jump Boots and Mid-Air Morph (Intermediate)
+          Hi-Jump Boots and Morph Glide (Intermediate)
```

Comment:
isnt this possible with just a good timed morph? this seems a tad high for a high glide.
(also you're missing the actual morph ball as an item here)

Reply:
I'm not able to make it with just a single morph, I just land on the spikes. In both directions.